client/verta/tests/pipeline/test_orchestrator.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
```

Comment:
I spent about 6 hours trying to write unit tests for the orchestrator and step handler, and it just wasn't happening 😭 mainly because they involve fetching artifacts, which would've required

- mocking 6 HTTP endpoints, which was awful to debug
- somehow overriding the client's artifact download caching behavior, because otherwise the get-artifact HTTP sometimes calls don't get made, which causes `responses` to fail the test

80% of the >100 lines of test code was just setup, which I'd hate to have to read and maintain.